molecule/system_settings_logging/converge.yml

```
@@ -0,0 +1,28 @@
+---
+- name: converge
+  hosts: all
+  become: true
+  tasks:
+    - name: Converge - Set preserve logs
+      puzzle.opnsense.system_settings_logging:
+        preserve_logs: 10
+
+    - name: Converge - Set max log file size
+      puzzle.opnsense.system_settings_general:
```

This should probably be the `system_settings_logging` module as well:
```suggestion
      puzzle.opnsense.system_settings_logging:
```